pkg/velero/server/args.go

```diff
@@ -39,6 +39,17 @@ func (a Args) StringArr(dpaFeatureFlags []string, logLevel string) ([]string, er
 	} else if boolptr.IsSetToFalse(a.DefaultVolumesToFsBackup) {
 		args = append(args, "--default-volumes-to-fs-backup=false")
 	}
+	/**
+		// Commented out, please see:
+		//    https://github.com/openshift/oadp-operator/pull/1102#discussion_r1286054062
+		if a.UploaderType != "" {
+		uploaderType := strings.ToLower(a.UploaderType)
+		if uploaderType != "kopia" && uploaderType != "restic" {
+			return []string{}, fmt.Errorf("invalid uploader type, the supported values are 'restic' or 'kopia': %s", a.UploaderType)
+		}
+		args = append(args, fmt.Sprintf("--uploader-type=%s", uploaderType))
+	}
+	*/
```

Comment:
Instead of commenting out, lets just remove these lines. I don't think we'll ever want to configure UploaderType here.

Reply:
removed.